STYLE_GUIDE.md

```diff
@@ -0,0 +1,25 @@
+# DELV Solidity Style Guide
+
+## Commenting
+
+1. Use complete sentences and end them with the appropriate puntuaction (typically a period).
+2. Use correct spelling.
+3. Use correct capitalization.
+4. Comments should not go past the 80th column from the start of the line.
```

Comment:
I think this hard limit may become pretty irritating for longer-form commentary and math equations.

One alternative I like is prettier's [print width](https://prettier.io/docs/en/options.html#print-width) approach:

> In code styleguides, maximum line length rules are often set to 100 or 120. However, when humans write code, they don’t strive to reach the maximum number of columns on every line. Developers often use whitespace to break up long lines for readability. In practice, the average line length often ends up well below the maximum.
> 
> Prettier’s printWidth option does not work the same way. It is not the hard upper allowed line length limit. It is a way to say to Prettier roughly how long you’d like lines to be. Prettier will make both shorter and longer lines, but generally strive to meet the specified printWidth.
> 
> Remember, computers are dumb. You need to explicitly tell them what to do, while humans can make their own (implicit) judgements, for example on when to break a line.

Reply:
We already do this throughout the codebase. I will add an exception for math, but otherwise, we’ve been getting along fine with this.

Reply:
Yeah wow after poking around and checking column numbers this is actually upheld super well.... expected there to be way more lines in the 80->90 col range

Reply:
Yeah, if you do vertical splits on a 16 inch screen, it becomes very nice/helpful to follow this convention. Anything much larger will start to wrap.